queue/queue.py

```diff
@@ -1,5 +1,8 @@
 import sys
-sys.path.append('../linked_list')
+import os
+
+sys.path.append(f'{os.getcwd()}/linked_list')
+
```

Comment:
Have you tried ?

import sys
sys.path.append('../linked_list')
from linked_list import LinkedList

Reply:
I did, it wasn't working for me - I'm still unsure why. I didn't dig too deep though.